docs/home/contribute/style-guide.md

```diff
@@ -172,6 +172,17 @@ apiVersion: v1 # earlier versions use...
 kind: Pod
 ...
 ```
+## Accessibility Guidelines
+By law in many countries, content must be accessible to people who have disabilities, such as visual impairments. Anything that the reader must know or do needs to be in text, too, not just in images.
+
+<table>
+  <tr><th>Content</th><th>Usage</th></tr>
+  <tr><td>Illustrations</td><td> Include figure tags &lt;figure&gt; and alternative text tags &lt;alt&gt;. Introduce illustrations or code listings with a clear lead-in sentence. Provide detailed instructions and explanations after the image or code example.</td></tr>
+  <tr><td>Code</td><td>Do not use screen captures or other graphics to show code, commands, or screen output that readers must do something with or know about. Ensure that all code is in code font so that it is readable by software for the vision-impaired.</td></tr>
+  <tr><td>Tables</td><td>Avoid rowspan or colspan (span tags) if possible. Use simple tables.</td></tr>
```

Comment:
These last two bullets seem to be related to accessibility in general rather than just as it applies to images, which is the focus of most of the top. I think this might be better laid out in two sections - one related to images, and another related to more general content and making it more accessible to screen readers, etc.

There's some sections about "don't use latin acronyms" that also appears to apply, and I wonder if we shouldn't consider re-organizing the doc to bring those together with these later suggestions.